lib/types.d.ts

```diff
@@ -157,3 +157,5 @@ export interface IssueDataViewInput {
   // isRootIssueLoading: boolean;
   // isPendingChildrenLoading: boolean;
 }
+
+export type BrowserMetricsProvider = import('@ipfs-shipyard/ignite-metrics').BrowserMetricsProvider
```

Comment:
safely getting the type here since importing it is sketchy with nextjs while this package is still cjs.